services/movie.py

```diff
@@ -1,14 +1,19 @@
 from django.db.models import QuerySet
+from django.db import transaction
 
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
 ) -> QuerySet:
```

Comment:
You should specify inner type of `QuerySet`. Fix everywhere, too
```suggestion
) -> QuerySet[Movie]:
```

Reply:
do it